wsdl/ver10/schema/onvif.xsd

```diff
@@ -1124,6 +1124,13 @@ IN NO EVENT WILL THE CORPORATION OR ITS MEMBERS OR THEIR AFFILIATES BE LIABLE FO
 					<xs:documentation>the maximum output bitrate in kbps</xs:documentation>
 				</xs:annotation>
 			</xs:element>
+			<xs:element name="AverageBitrate" type="xs:int">
```

Comment:
We use "AverageBitRateSupported", but here we use "AverageBitrate".
Can we stick with one type of usage, either "BitRate" like CBR, or "Bitrate"

Reply:
addressed feedback